app/views/courses/preview/financial_support/_scholarship_and_bursary.html.erb

```diff
@@ -1,26 +1,34 @@
+<% if course.has_financial_incentives? %>
   <p class="govuk-body">
-    With a scholarship or bursary, you’ll also get early career payments of £5,000 each in your third and fifth year of teaching (£7,500 in <%= link_to 'some areas of England', 'https://www.gov.uk/guidance/mathematics-early-career-payments-guidance-for-teachers-and-schools', class: "govuk-link" %>).
+    You could be eligible for either:
   </p>
-<% end %>
 
-<p class="govuk-body">
-  To qualify for a scholarship you’ll need a degree of 2:1 or above in <%= course.subjects.first %> or a related subject. For a bursary you’ll need a 2:2 or above in any subject.
-</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <% if course.has_scholarship? %>
+      <li data-qa="course__scholarship_amount">a scholarship of <%= number_to_currency(course.scholarship_amount) %></li>
+    <% end %>
+    <% if course.has_bursary? %>
+      <li data-qa="course__bursary_amount">a bursary of <%= number_to_currency(course.bursary_amount) %></li>
+    <% end  %>
+  </ul>
 
-<p class="govuk-body">
-  You can’t claim both a bursary and a scholarship - you can only claim one.
-</p>
+  <% if course.has_early_career_payments? %>
+    <div data-qa="course__early_career_payment_details">
+      <p class="govuk-body">
+        With a scholarship or bursary, you’ll also get early career payments of £2,000 in your second, third and fourth years of teaching (£3,000 in <%= link_to 'some areas of England', 'https://www.gov.uk/guidance/mathematics-early-career-payments-guidance-for-teachers-and-schools', class: "govuk-link" %>).
+      </p>
 
-<p class="govuk-body">
-  Find out how to <%= link_to 'apply for a scholarship', 'https://getintoteaching.education.gov.uk/funding-and-salary/overview/scholarships', class: 'govuk-link' %>. You don’t need to apply for a bursary - if you’re eligible, you’ll automatically start receiving it once you begin your course. Find out <%= link_to 'how you’ll be paid', 'https://getintoteaching.education.gov.uk/funding-and-salary/overview/how-you-will-be-paid', class: 'govuk-link' %>.
-</p>
+      <p class="govuk-body">
+        To qualify for a scholarship you’ll need a degree of 2:1 or above in <%= course.subject_name %> or a related subject. For a bursary you’ll need a 2:2 or above in any subject.
+      </p>
+
+      <p class="govuk-body">
+        You can’t claim both a bursary and a scholarship - you can only claim one.
+      </p>
+
+      <p class="govuk-body">
+        Find out how to <%= link_to 'apply for a scholarship', 'https://getintoteaching.education.gov.uk/funding-and-salary/overview/scholarships', class: 'govuk-link' %>. You don’t need to apply for a bursary - if you’re eligible, you’ll automatically start receiving it once you begin your course. Find out <%= link_to 'how you’ll be paid', 'https://getintoteaching.education.gov.uk/funding-and-salary/overview/how-you-will-be-paid', class: 'govuk-link' %>.
+      </p>
+    </div>
+  <% end %>
```

Comment:
https://github.com/DFE-Digital/search-and-compare-ui/blob/e26b23f3b803be59e82c1f0763402d01b1713bd9/shared/Views/Shared/Components/CourseDetails/FinancialSupport/_FinancialSupport.cshtml#L29-L32

![image](https://user-images.githubusercontent.com/470137/69143354-aebf6980-0ac0-11ea-90b5-0abfcecd2323.png)

Other Loans??

https://github.com/DFE-Digital/search-and-compare-ui/blob/e26b23f3b803be59e82c1f0763402d01b1713bd9/shared/Views/Shared/Components/CourseDetails/FinancialSupport/_FinancialSupport.cshtml#L35

![image](https://user-images.githubusercontent.com/470137/69143036-fbef0b80-0abf-11ea-80c8-332875c3051b.png)

what happens if you outside uk